sources/install/package_desktop.sh

```diff
@@ -0,0 +1,105 @@
+#!/bin/bash
+# Author: The Exegol Project
+
+source common.sh
+
+set -e
+
+function install_xfce() {
+    # CODE-CHECK-WHITELIST=add-aliases,add-history,add-test-command,add-to-list
+    colorecho "Installing and configuring xfce desktop"
+
+    # DEBUG TOOLS
+    # TODO remove
+    fapt terminator firefox-esr iproute2
+
+    # Dependencies
+    fapt tigervnc-standalone-server tigervnc-xorg-extension tigervnc-viewer novnc websockify xfce4 dbus-x11 intltool libtool tigervnc-tools
+
+    # Icons
+    fapt librsvg2-common papirus-icon-theme
+
+    # VNC
+    mkdir ~/.vnc
+    cp /root/sources/assets/desktop/configuration/xstartup.conf ~/.vnc/xstartup
+    chmod u+x ~/.vnc/xstartup
+
+    # Theme
+    mkdir /root/.themes
+    cp /root/sources/assets/desktop/Prof_XFCE_2_1.tar.gz /tmp
```

Comment:
Why cant we download this archive ? Is it custom or not ?

Reply:
answered before (will see later when we take care of all desktop's leftover TODOs)